### PR TITLE
feature/ansible

### PR DESCRIPTION
**Pull request by 0xBienCuit**
Basically did 3 little tasks as this is my first time contributing, but nevertheless in college I worked on implementing something similar and felt motivated to contribute.

The 3 tasks basically consist of the follolwing:

1. Trying to adhere to coding standard by replacing the hardcoded values with variables in `ansible/roles/google-chrome`.
2. Added error handling and retries for tasks that are susceptible to fail due to network related issues in `ansible/roles/nmap/tasks/nmap.yml`
3. Did some optimization on the tasks in `ansible/roles/nfs-server/tasks/main.yml` to reduce number of steps by combining the directory creation and ACL setting into a single block. Reduces the number of steps. 

#### Changed
- Refactored directory creation and ACL setting into a single block for better readability and maintainability in `ansible/roles/nfs-server/tasks/main.yml`
#### Fixed
- Ensured idempotency by using Ansible's `loop` feature for directory creation and ACL setting in `ansible/roles/nfs-server/tasks/main.yml`

#### Improved
- Improved error handling by adding retries to tasks that might fail due to transient issues in `ansible/roles/nmap/tasks/nmap.yml`.
- Enhanced security by ensuring sensitive data is handled securely in `ansible/roles/nfs-server/tasks/main.yml` .

#### Removed

- Removed hardcoded values and replaced them with variables for better configurability and reusability in `ansible/roles/nfs-server/tasks/main.yml` .

#### Added

- `ansible/roles/google-chrome/vars/main.yml`
- Added a task to ensure the `redteam` group exists in `ansible/roles/nfs-server/tasks/main.yml

### CODE REVIEW

File: ansible/roles/google-chrome/tasks/main.yml [hunk 1]
```diff
@@ -26,4 +29,4 @@
   copy:
     src: files/chrome-policies.json
     dest: /etc/opt/chrome/policies/managed/policies.json
-    mode: 0644
+    mode: 0644<
```
Comment (tophertimzen) [inline ansible/roles/google-chrome/tasks/main.yml:32]: Accidental "<" here?

Comment (0xBienCuit) [reply ansible/roles/google-chrome/tasks/main.yml:32]: Yeah good catch, will correct that!

**Comment by 0xBienCuit** [comment]
@tophertimzen  fixed the typo

**Review by willk — APPROVED** [review]
lgtm, thanks for the pr

**Pull request merged**